apis/core.oam.dev/common/types.go

```diff
@@ -542,16 +422,13 @@ type ClusterPlacement struct {
 	Distribution Distribution `json:"distribution,omitempty"`
 }
 
-// ResourceCreatorRole defines the resource creator.
-type ResourceCreatorRole string
```

Comment:
why this need to be changed? Can it be removed to workflow repo?

Reply:
We can use string directly instead.

Reply:
the string type lose the constraint

Reply:
If we want the constraint, we need to add the constraint to the workflow repo but I don't think it's appropriate.

Reply:
ok